### PR TITLE
Comment media fe

### PR DESCRIPTION
**Pull request by NeuralAlchemist**
- Added FE functionality for media upload inside comment form
Screenshot 2021-05-15 at 12 13 39

## User Story 39-3 

As a pet owner, I want to create a comment with images, so that I can comment images

## User Story 67-3

As user of Cuddly, I want to create a comment with videos, so that I can share content about my pet

### CODE REVIEW

File: frontend/src/components/comments/CommentCard.jsx [hunk 1]
```diff
@@ -107,6 +112,21 @@ export default function CommentCard({
...
+      {hasImage && (
+        <img
+          src={`data:${comment.imageType};base64, ${comment.image}`}
+          height="100%"
+          width="100%"
+        />
+      )}
+      {hasVideo && (
+        <ReactPlayer
+          url={require(`../../videos/${comment.videoName}`)}
```
Comment (eeels22) [inline frontend/src/components/comments/CommentCard.jsx]: Please amend the file path to include the comment folder

`url={require(`../../videos/comment/${comment.videoName}`)}`

File: frontend/src/components/comments/CommentForm.jsx [hunk 2]
```diff
@@ -1,24 +1,47 @@
...
+export default function CommentForm({ post, onSubmit, onSubmitMedia }) {
   // Local State
-  const [commentContentText, setCommentContentText] = useState('');
+  const [contentText, setContentText] = useState("");
   const [length, setLength] = useState();
-  const commentURL = require('../../assets/images/comment.svg');;
-
+  const [contentFile, setContentFile] = useState();
+  const [isFilePicked, setIsFilePicked] = useState(false);
   // Methods
```
Comment (eeels22) [inline frontend/src/components/comments/CommentForm.jsx:14]: please add an empty line above // Methods

File: frontend/src/components/comments/CommentCard.jsx (hunk 1 above)
Comment (NeuralAlchemist) [reply frontend/src/components/comments/CommentCard.jsx]: fixed

File: frontend/src/components/comments/CommentForm.jsx (hunk 2 above)
Comment (NeuralAlchemist) [reply frontend/src/components/comments/CommentForm.jsx:14]: fixed

**Comment by eeels22** [comment]
Still getting this error until I refresh

Screenshot 2021-05-15 at 12 55 50

File: frontend/src/components/FormFooter.jsx [hunk 3]
```diff
@@ -19,7 +19,7 @@ export default function FormFooter({
   }
   return (
     <>
-      <div className="media-name-character-limit-display">
+      <div className={`media-name-character-limit-display ${buttonText}`}>
```
Comment (eeels22) [inline frontend/src/components/FormFooter.jsx:22]: Is the `buttonText` props used in css styling? Do we need it as a className?

**Review by eeels22 — APPROVED** [review]
Approving since the video error is an existing bug

**Review by deciocandido — APPROVED** [review]
Looks really good

**Pull request merged**